website/src/components/Footer/index.tsx

```diff
@@ -136,55 +136,93 @@ const Footer: React.FC = () => {
+                "to": `${getdatesForContribution().today}`,
+                "raw": {
+                    "from": "now-1m",
+                    "to": "now"
+                }
+            },
+            "from": `${getdatesForContribution().priorDateTimestamp}`,
+            "to": `${getdatesForContribution().todayTimestamp}`
+        }
+        
         //getting the top contributors from github by sending the api order as desc
-
         useEffect(() => {
-           fetch(githubApiContributors)
-           .then((res) => res?.json())
-           .then(
-             (result) => {
-               setIsLoaded(true);
-               setItems(result);
-             },
-             (error) => {
-               setIsLoaded(true);
-               console.error(error);
-             }
-           );
-          return () => {
-            setItems([]);
-          };
-        }, []); // eslint-disable-line react-hooks/exhaustive-deps
+            fetch(githubApiContributors, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(reqBodyToFetchContributors),
+            })
+                .then(response => response.json())
+                .then(contributors => {
+                    setIsLoaded(true);
+                    setItems(contributors?.results?.A?.frames[0]?.data?.values[1]);
+                })
+                .catch((error) => {
+                    console.error('Error:', error);
+                });
+        });
+
+  
+
 
         return (
-          <div>
-            {isLoaded && items.length && (
-              <>
-                 <Typography variant='h6' className={classes.columnTitle}>
-                    {t('footer.topContributors')}
-                 </Typography>
-                <Typography className={classes.columnListWrapper}>
-                  {items?.slice(0, 3).map((item: any) => {
-                    return (
-                        <Link
-                          href={item.html_url}
-                          target="_blank"
-                          className={classes.columnListItem} key={item?.login}
-                        >
-                          {item.login}
-                        </Link>
-                    );
-                  })}
-                </Typography>
-              </>
-            )}
-          </div>
+            <div>
+                {isLoaded && items?.length && (
+                    <>
+                        <Typography variant='h6' className={classes.columnTitle}>
+                            {t('footer.topContributors')}
+                        </Typography>
+                        <Typography className={classes.columnListWrapper}>
+                            {items?.slice(0, 3).map((item: any) => {
+                                return (
+                                    <Link
+                                        href={item.html_url}
```

Comment:
we are only getting the username right!?

```suggestion
                                        href={`https://github.com/${item}`}
```